configs/typescript-config-spirit/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "typescript-config-spirit",
+  "description": "TypeScript configuration for Spirit Design System",
+  "version": "0.0.0",
```

Comment:
v0.1.0 (the lowest correct semver version AFAIK) will be released automatically?

Reply:
Yeah, will be re-versioned with the next publish. However, we can start even with the `1.0.0`.